custom/covid/tasks.py

```diff
@@ -46,38 +44,6 @@ def prime_formplayer_dbs():
             prime_formplayer_db_for_user.delay(domain, row[0], row[1], task_cutoff_hour=TASK_WINDOW_CUTOFF_HOUR)
 
 
-@no_result_task(queue='ush_background_tasks', max_retries=3, bind=True, rate_limit=RATE_LIMIT)
-def prime_formplayer_db_for_user(self, domain, request_user_id, sync_user_id,
```

Comment:
FWIW it looks like there are no real domains on the feature flag anymore.  Just added a tech debt ticket to remove it
https://dimagi.atlassian.net/browse/USH-5810
Could probably do that as part of this PR if you feel inspired

Reply:
I would prefer to have that sort of clean up in another branch, just to keep this PR more focused. I made another branch for the clean up `jt/clean-up-covid-code` https://github.com/dimagi/commcare-hq/compare/jt/clean-up-covid-code?expand=1 I referenced it in the description of the ticket you made